.pre-commit-config.yaml

```diff
@@ -0,0 +1,32 @@
+- repo: local
```

Comment:
So we'll want to add directions to the README.md file on how to enable pre-commit hooks. You can just copy the text at the bottom of the file here under "Contributing"
https://github.com/RTIInternational/SMART/tree/CTP_improvements